app/(app)/feedback.tsx

```diff
@@ -1,20 +1,124 @@
 import MainContainer from '../../components/container/MainContainer';
+import FeedbackSentModal from '../../components/modals/FeedbackSentModal';
+import { typography } from '../../config/typography';
 
-// TODO: implement feedback page
-/**
- * TASKS:
- * Implement feedback page
- * Should include a reactions component, a textarea and button
- */
+export type TypeRatingStates = {
+  icon: string;
+  text: string;
+  value: number;
+};
+
+const Feedback = () => {
+  const { colors } = useTheme();
+  const router = useRouter();
+  const [showModal, setShowModal] = useState(false);
+  const [selectedRating, setSelectedRating] = useState(2);
+  const [description, setDescription] = useState('');
+  const ratingStates: Array<TypeRatingStates> = [
+    { icon: '😔', text: 'Bad', value: 0 },
+    { icon: '😐', text: 'Okay', value: 1 },
+    { icon: '😊', text: 'Great', value: 2 },
+  ];
+  const openModal = () => {
+    setShowModal(true);
+  };
 
-const feedback = () => {
   return (
-    <MainContainer preset="scroll">
-      <StyledText>feedback</StyledText>
+    <MainContainer preset="scroll" safeAreaEdges={['top']}>
+      <FeedbackSentModal showModal={showModal} />
+      <View style={styles.FeedBackBannerParent}>
+        <FeedBackBanner />
+        <BackNavigationButton text="Feedback" onPress={() => router.back()} />
+      </View>
+      <View style={styles.FeedBackFormContainer}>
+        <StyledText size="lg" font="bold" variant="text" style={[styles.FeedBackFormTitle, { color: colors.primary }]}>
+          Your feedback helps us improve
+        </StyledText>
+        <Space size={29} />
+        <View style={[styles.FeedBackForm, { backgroundColor: colors.background, borderColor: colors.border }]}>
+          <StyledText style={styles.FeedBackFormLabel} size="base">
+            How is/was the event
+          </StyledText>
+          <View style={styles.FeedBackFormRatingContainer}>
+            {ratingStates.map((rating, index) => {
+              return (
+                <FeedBackRatingButton
+                  rating={rating}
+                  key={index}
+                  onPress={setSelectedRating}
+                  onSelected={rating.value === selectedRating}
+                />
+              );
+            })}
+          </View>
+        </View>
+        <Space size={30} />
+        <TextInput
+          style={[
+            styles.feedbackInput,
+            { backgroundColor: colors.bg, borderColor: colors.borderColor, color: colors.bgInverse },
+          ]}
+          placeholder="Type message here"
+          placeholderTextColor={colors.placeHolder}
+          value={description}
+          onChangeText={setDescription}
+        />
+        <Space size={26} />
+        <SubmitFeedbackButton openModal={openModal} text="SUBMIT FEEDBACK" />
+      </View>
     </MainContainer>
   );
 };
 
-export default feedback;
+const styles = StyleSheet.create({
+  FeedBackBannerParent: {
```

Comment:
Quick question, do we have any specific naming convention we prefer when it comes to styling? I know some devs like `PascalCase` and commonly we use `camelCase`

Reply:
Fixed in commit 755390cc68c4b0274166b33afa11a7db03e11116